src/components/adaptive-cards/Component/Component.jsx

```diff
@@ -166,9 +169,19 @@ export default function Component({
       case acPropTypes.targetElements:
       case acPropTypes.text:
       case acPropTypes.tooltip:
+      case acPropTypes.title:
       case acPropTypes.type:
       case acPropTypes.url:
+        break;
       case acPropTypes.value:
+      case acPropTypes.valueOff:
+      case acPropTypes.valueOn:
+        compData = {
```

Comment:
why is this needed?

Reply:
the linter doesn't allow us to directly mutate the values passed as parameters